ldk/node/src/hostClientFacade.ts

```diff
@@ -36,21 +39,25 @@ export default class HostClientFacade implements HostServices {
 
   public browser: BrowserClient = new BrowserClient();
 
+  constructor(logger: Logger) {
+    this.logger = logger;
+  }
+
   public connect(
     connInfo: ConnInfo.AsObject,
     session: Session.AsObject,
   ): Promise<void[]> {
     return Promise.all([
-      this.whisper.connect(connInfo, session),
-      this.storage.connect(connInfo, session),
-      this.keyboard.connect(connInfo, session),
-      this.clipboard.connect(connInfo, session),
-      this.cursor.connect(connInfo, session),
-      this.hover.connect(connInfo, session),
-      this.fileSystem.connect(connInfo, session),
-      this.process.connect(connInfo, session),
-      this.window.connect(connInfo, session),
-      this.browser.connect(connInfo, session),
+      // this.browser.connect(connInfo, session, this.logger),
+      // this.hover.connect(connInfo, session, this.logger),
+      // this.window.connect(connInfo, session, this.logger),
```

Comment:
Safe to remove these?

Reply:
I'm actually going to comment them, and their corresponding properties out. We haven't implemented those services yet but we should being doing that sometime soon.